duckduckgo/macos-browser
# Improvements in CPM coverage

Task/Issue URL: https://app.asana.com/0/0/1202900383044303/f
Product Review: https://app.asana.com/0/1142021229838617/1202892370714490/f
Tech Design URL:
CC:

**Description**:

This update adds support for a bunch of new CMPs, increasing the CPM coverage



**Steps to test this PR**:
1. Enable cookie prompt management
2. Browse some sites from the list in https://app.asana.com/0/1142021229838617/1202892370714490/f
3. Cookie popups should be rejected automatically

**Testing checklist**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
**When ready for review, remember to post the PR in MM**
